libs/dfi/src/dyn_type.c

```diff
@@ -731,7 +732,7 @@ void dynType_deepFree(dyn_type *type, void *loc, bool alsoDeleteSelf) {
                 //nop
                 break;
             default:
-                LOG_ERROR("Unexpected switch case. cannot free dyn type %c\n", type->descriptor);
+                celix_err_pushf("Unexpected switch case. cannot free dyn type %c\n", type->descriptor);
```

Comment:
Why not `#define LOG_ERROR celix_err_pushf`?

Reply:
agree